src/websocket/broadcast/gameBroadcast.ts

```diff
@@ -164,9 +164,14 @@ export const broadcastInGameUpdate = async (
   return usersInGame;
 };
 
-export const broadcastGameStart = async (ws: WebSocketClient, gameId: string, usersInGame: User[]): Promise<void> => {
+export const broadcastGameStart = async (
+  ws: WebSocketClient,
+  gameId: string,
+  hostConnectionId: string,
+  usersInGame: User[],
+): Promise<void> => {
   const connectionIds = getConnectionIdsFromUsers(usersInGame);
-  const { hands } = await initGameState(gameId, connectionIds);
+  const { hands } = await initGameState(gameId, hostConnectionId, connectionIds);
```

Comment:
What is the function of hostConnectionId? I don't see it being used in initGameState

Reply:
nice catch 😃 IDE didn't catch this parameter is not being used haha. Gonna remove it and fix tests